src/protocol.h

```diff
@@ -291,6 +291,9 @@ enum ServiceFlags : uint64_t {
     // See BIP159 for details on how this is implemented.
     NODE_NETWORK_LIMITED = (1 << 10),
 
+    // NODE_P2P_V2 means the node supports BIP324 transport
+    NODE_P2P_V2 = (1 << 11),
```

Comment:
in commit 1768aff56d6feedcb22b21df12e8926924fdac6a: this flag should also be added to the `ALL_SERVICE_FLAGS` array in `test/util/net.h`, I guess (used by fuzz tests):
https://github.com/bitcoin/bitcoin/blob/8f7b9eb8711fdb32e8bdb59d2a7462a46c7a8086/src/test/util/net.h#L61-L68